lib/migration-payloads/validation/schema-validation.js

```diff
@@ -55,8 +55,12 @@ const itemsValid = Joi.object().keys({
 
 const fieldSchema = Joi.object().keys({
   id: Joi.string().required(),
+  newId: Joi.string()
+    .invalid(Joi.ref('id'))
+    .max(64)
+    // the empty case will be caught by joi by default, we don't want duplicate errors
+    .regex(/^$|^[a-zA-Z][a-zA-Z0-9_]*$/),
```

Comment:
I tried using joi's `string.allow('')` method, but then neither `regex(..)` nor `min(1)` get applied and an empty string is considered valid...